scripts/itest_part.sh

```diff
@@ -11,6 +11,13 @@ NUM_TRANCHES=$2
 shift
 shift
 
+# Start postgres
+docker rm -f lnd-postgres
+docker run --name lnd-postgres -e POSTGRES_PASSWORD=postgres -p 6432:5432 -d postgres
```

Comment:
Should only be started if the backend variable is set to postgres to avoid unnecessary docker pulls on Travis (running into the limit some times already).

Reply:
If statement added